### ISSUE TITLE
svt-vp9 support

### ISSUE THREAD

**Issue by yulin-li** [BtbN/FFmpeg-Builds#376]
Is it possible to add an optional [svt-vp9](https://github.com/OpenVisualCloud/SVT-VP9/blob/master/ffmpeg_plugin/README.md) support?

**Comment by BtbN** [comment]
I don't think FFmpeg supports svt-vp9, does it?

**Comment by yulin-li** [comment]
> I don't think FFmpeg supports svt-vp9, does it?

No, FFmpeg doesn't support it officially, but the `svt-vp9` repo provides a patch so it could be built as a plugin.

https://github.com/OpenVisualCloud/SVT-VP9/blob/master/ffmpeg_plugin/README.md

**Comment by BtbN** [comment]
I don't support any third party patches to ffmpeg in this repo.
Plus, maintaining them would be a pain, since ffmpeg is a rather moving target.
So, unless you get the patch upstreamed, there can't be svt-vp9 support.
